src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -3990,14 +3990,6 @@ void DoorLockServer::clearAliroReaderConfigCommandHandler(CommandHandler * comma
     EndpointId endpointID = commandPath.mEndpointId;
     ChipLogProgress(Zcl, "[ClearAliroReaderConfig] Incoming command [endpointId=%d]", endpointID);
 
-    // If Aliro Provisioning feature is not supported, return UNSUPPORTED_COMMAND.
-    if (!SupportsAliroProvisioning(endpointID))
```

Comment:
We need to remove [https://github.com/project-chip/connectedhomeip/blob/9d3d8d21c249410070d3275fe5900[…]984e19db9d/src/app/clusters/door-lock-server/door-lock-server.h](https://github.com/project-chip/connectedhomeip/blob/9d3d8d21c249410070d3275fe59003984e19db9d/src/app/clusters/door-lock-server/door-lock-server.h#L231) and [https://github.com/project-chip/connectedhomeip/blob/9d3d8d21c249410070d3275fe5900[…]984e19db9d/src/app/clusters/door-lock-server/door-lock-server.h](https://github.com/project-chip/connectedhomeip/blob/9d3d8d21c249410070d3275fe59003984e19db9d/src/app/clusters/door-lock-server/door-lock-server.h#L241)

Reply:
No, those are still used.